Structure_Engine/Compute/Integrate.cs

```diff
@@ -95,6 +105,11 @@ public static partial class Compute
 
         /***************************************************/
 
+        [Description("Calculates all non-torsional section constants for a section profile based its edge curves and adjusts the profile curves to make them centred around the origin.")]
+        [Input("profile", "The profile to integrate.")]
+        [Input("tolerance", "The angleTolerance for dividing the section curves.")]
+        [MultiOutput(0, "profile", "The profile used in the integration, with section curves updated to be centred around the global origin.")]
```

Comment:
The profile used in the integration. The section curves are translated to be centred around the global origin.